docs/source/learners/llm.rst

```diff
@@ -1,2 +1,63 @@
 Large Language Models
 ========================
+LLM-only learners leverage the power of large language models to perform ontology learning tasks
```

Comment:
Please, explain a bit the methodology, maybe adding a single paragraph would be great!